<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edgeWarpWdg</class>
 <widget class="QWidget" name="edgeWarpWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>edgeWarpWdg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>属性</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="bt_selectRoi">
          <property name="text">
           <string>选择区域</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QPushButton" name="bt_ok">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>确定选取</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="2">
         <widget class="QPushButton" name="bt_cancel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>取消选取</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QCheckBox" name="cb_refer">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>参考矩阵</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QCheckBox" name="cb_showRegionAfterDet">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>测后显示</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>图像比例尺：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="dsp_pixelSize">
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>当前边界：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QComboBox" name="cob_edgeIndex">
          <item>
           <property name="text">
            <string>底面边界</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>管脚侧面1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>管脚侧面2</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>对比度：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="sp_gradient">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string>脚尖对比度：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QSpinBox" name="sp_leadThre">
          <property name="minimum">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="bt_advance">
            <property name="text">
             <string>高级设置</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string> 边线偏移：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QSpinBox" name="sp_lineOffset">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="minimum">
           <number>-99</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="5">
         <widget class="QGroupBox" name="grp_advance">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>目标芯片朝向</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>5</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QRadioButton" name="rb_UpToDw">
                <property name="text">
                 <string>上-&gt;下</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rb_DwToUp">
                <property name="text">
                 <string>下-&gt;上</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rb_LfToRt">
                <property name="text">
                 <string>左-&gt;右</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rb_RtToLf">
                <property name="text">
                 <string>右-&gt;左</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0" rowspan="2">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string> 目标颜色</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <property name="spacing">
               <number>5</number>
              </property>
              <item row="0" column="0">
               <widget class="QRadioButton" name="rb_black">
                <property name="text">
                 <string>黑色</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QRadioButton" name="rb_white">
                <property name="text">
                 <string>白色</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0" colspan="2">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_34">
               <property name="text">
                <string>边界类型：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cob_pol">
               <item>
                <property name="text">
                 <string>双向</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>黑到白</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>白到黑</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QComboBox" name="cob_first">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>0</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string> 最大对比</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string> 第一条</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_35">
               <property name="text">
                <string>角度范围：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QSpinBox" name="sp_degRange">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_36">
                 <property name="text">
                  <string>度</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_37">
               <property name="text">
                <string> 噪点距离：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QSpinBox" name="sp_rmDist">
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_38">
               <property name="text">
                <string>迭代次数：</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="sp_iteratNum">
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_33">
               <property name="text">
                <string> 边界毛边：</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QSpinBox" name="sp_step">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="7" column="0" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>翘曲补偿值：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dsp_offset">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QComboBox" name="cob_inputPixel">
          <item>
           <property name="text">
            <string>填入比例尺</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传入3D比例尺</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>传入像素比例尺</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>定位误差范围</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dsp_errWarpMin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="cb_errWarp">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>管脚翘曲：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="dsp_errWarpMax">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="cb_errWarpDiff">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>共面性差异：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="dsp_errWarpDiffMin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="dsp_errWarpDiffMax">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_17">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
